addons/binding/org.openhab.binding.russoundserial/src/main/java/org/openhab/binding/russoundserial/handler/RussoundSerialBridgeHandler.java

```diff
@@ -0,0 +1,75 @@
+package org.openhab.binding.russoundserial.handler;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
+import org.eclipse.smarthome.core.types.Command;
+import com.thejholmes.russound.RussoundAction;
+import com.thejholmes.russound.RussoundCommander;
+import com.thejholmes.russound.RussoundZoneInfoListener;
+import com.thejholmes.russound.Zone;
+import com.thejholmes.russound.ZoneInfo;
+import com.thejholmes.russound.serial.SerialCommandReceiver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RussoundSerialBridgeHandler extends BaseBridgeHandler {
+  private final RussoundCommander commander;
+  private final SerialCommandReceiver commandReceiver;
+  private final ZoneListener zoneListener;
+
+  public RussoundSerialBridgeHandler(Bridge bridge, RussoundCommander commander,
+      SerialCommandReceiver commandReceiver, ZoneListener zoneListener) {
+    super(bridge);
+    this.commander = commander;
+    this.commandReceiver = commandReceiver;
+    this.zoneListener = zoneListener;
+  }
+
+  public RussoundCommander getCommander() {
+    return commander;
+  }
+
+  @Override public void initialize() {
+    super.initialize();
+    this.commandReceiver.start();
+  }
+
+  @Override public void dispose() {
+    this.commander.destroy();
+    this.commandReceiver.stop();
+    super.dispose();
+  }
+
+  @Override public void handleCommand(ChannelUID channelUID, Command command) {
```

Comment:
Please format is such that @Override has its own line, ie please format all the files you added with the Eclipse ESH profile